test/screenshot/fixture.scss

```diff
@@ -18,8 +18,9 @@
 @import url("https://fonts.googleapis.com/icon?family=Material+Icons");
 
 $test-trim-color: #abc123; // Value must match `TRIM_COLOR_CSS_VALUE` in `image-cropper.js`
+$test-grid-color: #dddddd;
+$test-fixture-width: 348px;
```

Comment:
Where is 348px coming from?

Reply:
It's wide enough to fit 2 columns of buttons, and narrow enough to fit in a mobile Chrome viewport without having extra whitespace on the right side of the cropped image.

I just played around in the Chrome Dev Tools until I found the best value 😛 

The width of the cells is also hard-coded. They're wide enough to fit most of our buttons, narrow enough to fit in two columns in mobile Chrome, and they show the right-most and bottom-most ruler line instead of cutting off.

I originally calculated the widths, but it was such a pain in the ass to update the formulas every time I changed the borders/margins of the grid cells, so I figured it would be simpler to just hard-code the value for now.